cilium/assets/configuration/spec.yaml

```diff
@@ -4,9 +4,19 @@ files:
   options:
   - template: init_config
     options:
-      - template: init_config/openmetrics_legacy
+      - template: init_config/openmetrics
   - template: instances
     options:
+    - name: use_openmetrics
+      description: |
+        Use the latest OpenMetrics V2 implementation for more features and better performance.
+
+        Note: To see the configuration options for the legacy OpenMetrics implementation (Agent 7.33 or older),
+        https://github.com/DataDog/integrations-core/blob/7.33.x/cilium/datadog_checks/cilium/data/conf.yaml.example
+      enabled: true
+      value:
+        type: boolean
+        example: true
```

Comment:
Should this be `false` by default so if people update they don't have any change in behavior?

Reply:
No, this only affects new customers installing the integration. When existing users upgrade, their conf.yaml is persisted and their configurations remain the same. The `use_openmetrics` option in the code will default to `false`

Reply:
For it to default to false you need to make it explicit, if no default is defined the example is used as default https://github.com/DataDog/integrations-core/pull/10840/files#diff-0cff7d100d9aa79c2d57a2b445028d056a5ca08ce33b78225f96e1f6b87af308R344